services/blockchain-app-registry/tests/unit/shared/metadataIndex.test.js

```diff
@@ -0,0 +1,284 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+/* eslint-disable mocha/max-top-level-suites */
+const {
+	appMetaObj: mockAppMetaObj,
+	tokenMetaObj: mockTokenMetaObj,
+} = require('../../constants/metadataIndex');
+
+const tempDir = `${__dirname}/temp/${mockAppMetaObj.networkType}/${mockAppMetaObj.chainName}`;
+const mockAppMetaPath = `${tempDir}/${require('../../../config').FILENAME.APP_JSON}`;
+const mockTokenMetaPath = `${tempDir}/${require('../../../config').FILENAME.NATIVETOKENS_JSON}`;
+
+beforeEach(() => jest.resetModules());
+
+describe('Test indexTokensMeta method', () => {
+	it('should index token meta in db when called with valid metadata object', async () => {
+		jest.mock('lisk-service-framework', () => {
+			const actual = jest.requireActual('lisk-service-framework');
+			return {
+				...actual,
+				MySQL: {
+					getTableInstance: () => ({
```

Comment:
Same for all the tests
```suggestion
					...actual.MySQL,
					getTableInstance: () => ({
```

Reply:
Applied.